src/ert/gui/tools/design_matrix/design_matrix_panel.py

```diff
@@ -0,0 +1,47 @@
+from typing import Optional
+
+import pandas as pd
+from qtpy.QtGui import QStandardItem, QStandardItemModel
+from qtpy.QtWidgets import QDialog, QTableView, QVBoxLayout, QWidget
+
+
+class DesignMatrixPanel(QDialog):
+    def __init__(
+        self, design_matrix_df: pd.DataFrame, parent: Optional[QWidget] = None
+    ) -> None:
+        super().__init__(parent)
+
+        self.setWindowTitle("Design matrix parameters viewer")
```

Comment:
Should we have the title as viewer or preview?

Reply:
What do you mean by title?

Reply:
Can we change:
`self.setWindowTitle("Design matrix parameters viewer")` to
`self.setWindowTitle("Design matrix parameters preview")`

Reply:
Right, from this it sounds you are fine with both:

> Should we have the title as viewer or preview?

Reply:
Yes, we can keep it as viewer 👌

Reply:
Just change it to: `f"Parameters from {filename}"`

Reply:
Even better :)